pkg/utils/asynciter.go

```diff
@@ -143,6 +142,13 @@ func (ic *AsyncItemsCursor) NextItem() (v3io.Item, error) {
 
 	// Read response from channel
 	resp := <-ic.responseChan
+
+	// Ignore 404s
+	if resp.Error != nil && strings.Contains(resp.Error.Error(), "404") {
```

Comment:
This is pretty awful... Can't we get the actual http code from `resp`?

Reply:
Need to fill feature request to HTTP-GO lib to return error codes for each HTTP call
Please check with @pavius if it's something he can do quickly

Reply:
He is too busy, we can open a PR with a fix by ourselves but not in this PR and i guess it's not *that* urgent

Reply:
Ok for now, but please add a TODO and see if you can restrict this condition further (so it doesn't match something like `1534324049`).